ts/import-log/DetailsTable.svelte

```diff
@@ -0,0 +1,93 @@
+<!--
+Copyright: Ankitects Pty Ltd and contributors
+License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+-->
+<script lang="ts">
+    import * as tr from "@tslib/ftl";
+
+    import IconButton from "../components/IconButton.svelte";
+    import VirtualTable from "../components/VirtualTable.svelte";
+    import TableCellWithTooltip from "./TableCellWithTooltip.svelte";
+    import { magnifyIcon } from "./icons";
+    import { getRows, showInBrowser } from "./lib";
+    import type { SummarizedLogQueues } from "./types";
+
+    export let summaries: SummarizedLogQueues[];
+
+    $: rows = getRows(summaries);
+</script>
+
+<details>
+    <summary>{tr.importingDetails()}</summary>
+    <VirtualTable
+        class="details-table"
+        itemHeight={50}
+        itemsCount={rows.length}
+        containerHeight={500}
```

Comment:
The static height could be a problem on mobile, especially in landscape. Would it mak esense to ensure it doesn't exceed 90% of the viewport height?

Reply:
I made it fill available height.